src/views/FineView.vue

```diff
@@ -191,20 +196,25 @@ const handlePickedDates = handleSubmit(async (values) => {
   userStore.users.forEach((user: any) => {
     userFullNameMap[user.id] = `${user.firstName} ${user.lastName}`;
   });
+  const deletedUsers = userStore.getDeletedUsers.map((u: UserResponse) => u.id);
+  const activeUsers = userStore.getActiveUsers.map((u: UserResponse) => u.id);
   eligibleUsers.value = result.data.map((item: any) => {
+
     const fullName = userFullNameMap[item.id];
 
     // Extract balances from the item
     const [firstBalance, secondBalance] = item.balances || [null, null];
 
     return {
       ...item,
+      // @ts-ignore
+      gewisId: (userStore.users.find((u: UserResponse) => u.id === item.id) as UserResponse).gewisId || undefined,
       fullName,
       // Assign first and last balance based on the first and second items in the balances array
       firstBalance,
       lastBalance: secondBalance,
     };
-  });
+  }).filter((u: any) => !deletedUsers.includes(u.id) && activeUsers.includes(u.id));
```

Comment:
Why is this not fixed in the backend but in the frontend?

Reply:
I want to do both, and for now the bug is not fixed in the backend yet.